rdtho2525/Refactor-Tractor
# Package setup

## What does this PR do?
 - Adds new configs
 
## How should it be tested?
 - Run server with `npm` start
 - Content should appear on: http://localhost:8080/
